assets/sass/_top-button.scss

```diff
@@ -0,0 +1,17 @@
+.top-button {
+    border-radius: 3px;
```

Comment:
Nearly all of these should be replaceable with Tailwind classes, and [we have a `transition` mixin](https://github.com/pulumi/docs/blob/1afa9b04ca14fec1e7aea10222194ee27bd82567/assets/sass/_mixins.scss#L1-L3) that you can use for that particular bit. We should only write CSS when we can't use Tailwind.

Reply:
Incidentally, the preview clearly shows that the button is not being styled:


Localhost:

![image](https://user-images.githubusercontent.com/274700/81966991-1fa69080-95cf-11ea-8d5e-adcacb82f2c6.png)

Preview:

![image](https://user-images.githubusercontent.com/274700/81966907-06054900-95cf-11ea-857c-a86370b8271f.png)

This is almost surely happening because PostCSS is removing the styles you've defined here, as they aren't being referenced in the markup. (We do this intentionally to keep our CSS from becoming bloated -- I believe this has been mentioned before.) [The PostCSS config is here](https://github.com/pulumi/docs/blob/6de64d4e4b0a8ae91c61ac2647472def1b8800c9/assets/config/postcss.config.js). You probably need to add an exception to the whitelist to make sure your styles are retained.